client/signup/jetpack-connect/install-step.jsx

```diff
@@ -15,83 +16,100 @@ import JetpackExampleConnect from './example-components/jetpack-connect';
 
 const NEW_INSTRUCTIONS_JETPACK_VERSION = '4.2.0';
 
-export default React.createClass( {
-	displayName: 'JetpackInstallStep',
+class JetpackInstallStep extends Component {
 
-	confirmJetpackInstalled( event ) {
+	static propTypes = {
+		currentUrl: PropTypes.string.isRequired,
```

Comment:
👍  7f5a9392b

Sounds good, this is actually what some components already do down the line.